kubernetes_platform/python/release.sh

```diff
@@ -0,0 +1,37 @@
+# run from within ./kubernetes_platform/python
```

Comment:
Adding `#!/bin/bash` and copyright info to the file?

Reply:
Done.